spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -232,6 +239,25 @@ def remove_stderr_file(self):
         except Exception:
             pass
 
+    @Slot()
+    def poll_stderr_file_change(self):
+        """Check if the stderr file just changed"""
+        try:
+            mtime = os.stat(self.stderr_file).st_mtime
+        except Exception:
+            return
+
+        if mtime == self.stderr_mtime:
+            return
+        self.stderr_mtime = mtime
+        try:
+            stderr = self._read_stderr()
+        except Exception:
+            stderr = None
+        if stderr:
+            self.shellwidget._append_plain_text(
+                '\n' + stderr, before_prompt=False)

```

Comment:
I think this breaks the mechanism we have to report kernel startup errors on the info widget. So perhaps this should be done only when there's an active kernel associated to the client.